build_windows.md

```diff
@@ -16,17 +16,187 @@ Pandoc could be installed from https://pandoc.org/installing.html When performin
 "Install for all users" (to put the binaries into "Program Files" directory).
 
 ## Install WiX Toolset v3 (is preinstalled in GitHub runner)
-The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/.  Installing it into a clean VM might require
+The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/. Installing it into a clean VM might require
 an additional installation of .NET Framework 3.5 in advance
 ([instructions for adding .NET Framework 3.5 via enabling the Windows feature](https://learn.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows#enable-the-net-framework-35-in-control-panel))
 
-## Install msys2
+## Building and running
+
+### Install git and go
+
+To build Podman, the [git](https://gitforwindows.org/) and [go](https://go.dev) tools are required. In case they are not yet installed,
+open a Windows Terminal and run the following command (it assumes that [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/) is installed):
+
+```
+winget install -e GoLang.Go Git.Git
+```
+
+:information_source: A terminal restart is advised for the `PATH` to be reloaded. This can also be manually changed by configuring the `PATH`:
+
+```
+$env:Path += ";C:\Program Files\Go\bin\;C:\Program Files\Git\cmd\"
+```
+
+### Enable Hyper-V (optional)
+
+Podman on Windows can run on the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/) or 
+on [Hyper-V](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) .
+To us Hyper-V use the following command:
```

Comment:
Suggest: `To use Hyper-V, enter the foolowing command`

Reply:
:thumbsup: